src/commands/apps-create.ts

```diff
@@ -0,0 +1,210 @@
+import * as clc from "cli-color";
+import * as ora from "ora";
+
+import * as Command from "../command";
+import * as getProjectId from "../getProjectId";
+import * as FirebaseError from "../error";
+import { AppPlatform, createAndroidApp, createIosApp, createWebApp } from "../appsCreate";
+import { prompt, promptOnce, Question } from "../prompt";
+import * as requireAuth from "../requireAuth";
+import * as logger from "../logger";
+
+function promptDisplayName(): Question {
```

Comment:
I think this would be better served as a constant (DISPLAY_NAME_QUESTION) rather than a function.

Reply:
Sure, done